libc/utils/docgen/api.yml

```diff
@@ -0,0 +1,107 @@
+ctype.h:
+  functions:
+    isalnum:
+      returns: int
+      params:
+        - int
+    isalpha:
+      returns: int
+      params:
+        - int
+    isblank:
+      returns: int
+      params:
+        - int
+  # TODO: fill me out!
+fenv.h:
```

Comment:
Rather than have one long long running file, perhaps it would be better to rely on a convention where we have individual yaml files, one per header such as ctype.h.yml and fenv.h.yml (or ctype.yml fenv.yml). Then the header name is implied by the file name, and we can keep the individual files shorter and perhaps better organized?

Reply:
done in https://github.com/llvm/llvm-project/pull/87682/commits/2c30d1984683defeae021e6f5f58c5edae9fff2d